test/onnx/test_op_consistancy.py

```diff
@@ -0,0 +1,365 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and aten operators.
+
```

Comment:
nit: a short note on how to use this, like what command prompt to use for testing selected ops, would be handy.

Reply:
Done